### PR TITLE
feat: Added matrix name to template increasing readability

### PR DESCRIPTION
**Pull request by AlexanderSehr**
## Description

Added matrix name to template increasing readability

Before:
image

After:
image

### CODE REVIEW

File: .github/workflows/avm.res.insights.action-group.yml [hunk 1]
```diff
@@ -71,7 +71,7 @@ jobs:
   #   Call reusable workflow   #
   ##############################
   call-workflow-passing-data:
-    name: "Module"
+    name: "Run"
```
Comment (eriqua) [inline .github/workflows/avm.res.insights.action-group.yml:74]: Shouldn't we have this change in all caller workflows?

**Review by eriqua — CHANGES_REQUESTED** [review]
Great improvement on readability, both for the compressed test case and replacing `Module` with `Run`. 

The only update I'd challenge is the matrix header. Can we keep `PSRule` and `Deploy` instead of `job_test_psrule` and `job_module_deploy`?

File: .github/workflows/avm.res.insights.action-group.yml (hunk 1 above)
Comment (AlexanderSehr) [reply .github/workflows/avm.res.insights.action-group.yml:74]: Yes

**Comment by AlexanderSehr** [comment]
> Great improvement on readability, both for the compressed test case and replacing `Module` with `Run`.
> 
> The only udate I'd challenge is the matrix header. Can we keep `PSRule` and `Deploy` instead of `job_test_psrule` and `job_module_deploy`?

@eriqua Unfortunately not. The change was not intended and seems to be automatically done once you add a name to the matrix job. The name is then used for the jobs while the Id is used for the matrix header. I did not find a way to avoid this 😕

**Comment by eriqua** [comment]
> > Great improvement on readability, both for the compressed test case and replacing `Module` with `Run`.
> > The only udate I'd challenge is the matrix header. Can we keep `PSRule` and `Deploy` instead of `job_test_psrule` and `job_module_deploy`?
> 
> @eriqua Unfortunately not. The change was not intended and seems to be automatically done once you add a name to the matrix job. The name is then used for the jobs while the 

Heartbreaking

File: .github/workflows/avm.res.insights.action-group.yml (hunk 1 above)
Comment (AlexanderSehr) [reply .github/workflows/avm.res.insights.action-group.yml:74]: done

**Review by eriqua — APPROVED** [review]
:ok:

**Pull request merged**